src/elements/input.ts

```diff
@@ -4,12 +4,11 @@ import { customElement, property } from 'lit/decorators.js'
 @customElement('hangman-input')
 export class HangmanInput extends LitElement {
   @property()
-  inputLetter: string = 'a'
-
   render() {
     return html` <input
+      id="letter_input"
```

Comment:
i think this `id` isn't used anywhere, let's get rid of it

Reply:
True